examples/1_example_python_operator/hello_world_dag.py

```
@@ -6,10 +6,10 @@
     return 'Hello world from first Airflow DAG!'
 
-dag = DAG('DAG-Hello-World', description='Hello World DAG',
+dag = DAG('DAG-Hello-World', description='Hello World DAG Coderabbit',
           schedule_interval='0 12 * * *',
           start_date=datetime(2017, 3, 20), catchup=False)
 
 hello_operator = PythonOperator(task_id='hello_task', python_callable=print_hello, dag=dag)
 
-hello_operator
\ No newline at end of file
+hello_operator
```

Ruff

Found useless expression. Either assign it to a variable or remove it.

(B018)